sympy/functions/elementary/trigonometric.py

```diff
@@ -117,10 +117,19 @@ def _peeloff_pi(arg):
             K, p = a.as_two_terms()
             if p is S.Pi and K.is_Rational:
                 break
+            K = a.coeff(S.Pi)
+            if K and K.is_rational:
+                break
+
     else:
         return arg, S.Zero
 
-    m1 = (K % S.Half)*S.Pi
+    z = S.One
```

Comment:
Rename this variable conviently because when other people will read code then they should find it more readable and understandable. Rename it according to its motive of its initialisation.

Reply:
it is just an initialiser variable, what should i rename it to?

Reply:
it is trying to store the end result of calculating the coeff mod S.Half

Reply:
Renamed the variable so that people find it more readable